### DISCUSSION TITLE
[GTK] TableLayout not invalidating

### DISCUSSION THREAD

**Discussion by Beyley** [picoe/Eto#2226] [Q&A]
Whenever i attempt to update a TableLayout (inside a DynamicLayout inside a extended Scrollable) after the constructor of my `Form`, even though the items are there when i add them (checked with a debugger), and i do a Suspend/ResumeLayout, it never draws the items on my screen
 
Form:
https://github.com/Beyley/Kettu/blob/568da33cd0e3cd95835182b384601394da620fd1/Kettu.BinaryReader/ReaderForm.cs#L34
Main Control:
https://github.com/Beyley/Kettu/blob/568da33cd0e3cd95835182b384601394da620fd1/Kettu.BinaryReader/ReaderControl.cs#L10
Thing which calls the code which adds the `TableRow`s:
https://github.com/Beyley/Kettu/blob/568da33cd0e3cd95835182b384601394da620fd1/Kettu.BinaryReader/OpenNewCommand.cs#L23

**Comment by cwensley** [comment]
TableLayout doesn't currently allow changing its content after created, so you need to recreate the TableLayout.  Invalidate() simply gets the control and children to redraw themselves.

**Reply by Beyley** [reply]
> TableLayout doesn't currently allow changing its content after created, so you need to recreate the TableLayout. Invalidate() simply gets the control and children to redraw themselves.

Even if i refactor my code to clear the dynamic layout, create a new table layout, add the elements, *then* add it to the dynamic layout again, it still doesnt seem to actually draw on screen
